### PR TITLE
Tkachuk Andrey Homework 1

### PR DESCRIPTION
**Pull request by exDron**

### CODE REVIEW

File: src/tkachuk.andrey/homework.ts [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+ */
+
+export const noEmployeesMessage = 'нет сотрудников';
+
+export function maxSalary(salaries: SalaryInfo): number | string {
+  if (Object.keys(salaries).length === 0) {
+    return noEmployeesMessage;
+  }
+
+  let sortedSalaries = Object.values(salaries).sort((prev: number, next: number) => {
```
Comment (LimarenkoDenis) [inline src/tkachuk.andrey/homework.ts]: add type for sortedSalaries

File: src/tkachuk.andrey/homework.ts [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+var salaries = {
+  "Вася": 100,
+  "Петя": 300,
+  "Даша": 250
+};
+
+// ... ваш код выведет "Петя"
+ */
+
+export const noEmployeesMessage = 'нет сотрудников';
```
Comment (LimarenkoDenis) [inline src/tkachuk.andrey/homework.ts]: pls declare noEmployeesMessage in maxSalary scope

**Comment by LimarenkoDenis** [comment]
@exDron good solutions, thanks for your PR, I approved, fix comments if u want or u can merge it

File: src/tkachuk.andrey/homework.ts (hunk 1 above)
Comment (exDron) [reply src/tkachuk.andrey/homework.ts]: Missed it) Done!

File: src/tkachuk.andrey/homework.ts (hunk 2 above)
Comment (exDron) [reply src/tkachuk.andrey/homework.ts]: Yeah, thank you. Moved.

**Pull request merged**